workers/unity/Packages/com.improbable.gdk.core/Systems/WorkerSystem.cs

```diff
@@ -61,6 +66,17 @@ public bool HasEntity(EntityId entityId)
             return EntityIdToEntity.ContainsKey(entityId);
         }
 
+        internal void GetMessages()
+        {
+            Diff = connectionHandler.GetMessagesReceived();
```

Comment:
Do we want this to be returned? As we are setting the reference to itself over and over since we never recreate the diff

Reply:
In a multi-threaded version the reference will be different. It's an implementation detail for now that the ref is the same each time.